nengo/spa/basalganglia.py

```diff
@@ -141,3 +143,36 @@ def add_dot_input(self, index, source, symbol, scale):
         with self.spa:
             nengo.Connection(output, self.input[index:index+1],
                              transform=transform, synapse=self.input_synapse)
+
+    def add_scalar_input(self, index, source):
+        """ Add a scalar input that will vary over time.
+
+        This would be used for such a thing as the ouput of the Compare module.
+
+        Parameters
+        ----------
+        index : int
+            the index of the action
+        source : Source
+            the module output to read from
+        symbol : Source
+            the semantic pointer to compute the dot product with
+        scale : float
+            a scaling factor to be applied to the result
+        """
+        output, _ = self.spa.get_module_output(source.name)
+        if(output.size_out != 1):
+            raise NotImplementedError("Only sources with a dimension"
+                                      "of 1 can be scalar inputs")
+
+        try:
+            scale = float(eval(source.transform.symbol))
+        except ValueError:
+            raise NotImplementedError("Transform can only be a scalar value"
+                                      "the value %s is invalid"
+                                      % source.transform.symbol)
+
+        with self.spa:
+            nengo.Connection(output, self.input[index:index+1],
```

Comment:
> I kinda thought that was a thing too, but pep8 actually suggests using spaces to group operators by order of execution. In a case like this, I'm fine either way. Whichever seems more readable, and I think what's there is quite readable.

Sweet!  I'm very glad to know that...  there's been a number of times where I've wanted to have some operators with spaces and some without (```a*b + c*d```) and I'd been forcing myself to not do that...   Good to know at least some of my style instincts are okay....  ;)